changelog.txt

```diff
@@ -16,6 +16,9 @@ that repo.
 ## New Scripts
 - `workorder-recheck`: resets the selected work order to the ``Checking`` state
 
+## Misc Improvements
+- `quickfort`: implement {Empty} keycode for use in quickfort aliases; useful for defining blank-by-default alias values
```

Comment:
I'd suggest this just for consistency (doesn't look like the braces cause any RST formatting issues, but previous mentions of `{Ctrl}/{Alt}` used code formatting)
```suggestion
- `quickfort`: implement ``{Empty}`` keycode for use in quickfort aliases; useful for defining blank-by-default alias values
```